### PR TITLE
kv/bulk: clean up BufferingAdder's logging

### PR DESCRIPTION
**Pull request by dt**
Release justification: debug logging.

### CODE REVIEW

**Comment by dt** [comment]
@aayushshah15 @adityamaru Note that I don't think this fully explains the observed regression, since we first observed it before I "fixed" the initial-splits conditional; prior to #77352, I believe we never called `createInitialSplits`, so the sstBatcher's own split "if this is the first flush, add a split" logic should have been making this first split.

File: pkg/kv/bulk/sst_batcher.go [hunk 1]
```diff
@@ -380,8 +380,12 @@ func (b *SSTBatcher) doFlush(ctx context.Context, reason int, nextKey roachpb.Ke
 				} else {
 					b.flushCounts.splitWait += reply.Timing.Split
 					b.flushCounts.scatterWait += reply.Timing.Scatter
-					if reply.Stats != nil {
-						b.flushCounts.scatterMoved += sz(reply.Stats.Total())
+					if reply.ScatteredStats != nil {
+						moved := sz(reply.ScatteredStats.Total())
+						b.flushCounts.scatterMoved += moved
+						if moved > 0 {
+							log.VEventf(ctx, 1, "starting split scattered %s in non-empty range %s", moved, reply.ScatteredSpan)
```
Comment (adityamaru) [inline pkg/kv/bulk/sst_batcher.go:387]: not a big deal but any reason this is vmodule 1, when the same logline in the buffering adder is 3?

**Review by adityamaru — APPROVED** [review]

File: pkg/kv/bulk/buffering_adder.go [hunk 2]
```diff
@@ -144,9 +147,10 @@ func (b *BufferingAdder) SetOnFlush(fn func(summary roachpb.BulkOpSummary)) {
 // Close closes the underlying SST builder.
 func (b *BufferingAdder) Close(ctx context.Context) {
 	log.VEventf(ctx, 2,
-		"bulk adder %s ingested %s, spent %v sorting and %v flushing (%v sending, %v splitting, %v scattering %v). Flushed %d due to buffer (%s) size. Flushed chunked as %d files (%d after split-retries), %d due to ranges, %d due to sst size.",
+		"bulk adder %s ingested %s, spent %v filling %v sorting and %v flushing (%v sending, %v splitting, %v scattering %v). Flushed %d due to buffer (%s) size. Flushed chunked as %d files (%d after split-retries), %d due to ranges, %d due to sst size.",
 		b.name,
 		sz(b.sink.totalRows.DataSize),
+		b.flushCounts.totalFilling,
 		b.flushCounts.totalFlush,
 		b.flushCounts.totalSort,
```
[the remark this answers is not in the document]
Comment (dt) [reply pkg/kv/bulk/buffering_adder.go]: It was / is on master. The 2nd commit here fixes it, doesn't it?

File: pkg/kv/bulk/sst_batcher.go (hunk 1 above)
Comment (dt) [reply pkg/kv/bulk/sst_batcher.go:387]: made them both 1.

**Pull request merged**